src/containers/LeftNav/index.js

```diff
@@ -98,23 +99,24 @@ export const LeftNav = (props) => {
     <NavWrapper>
       { navVisible && <React.Fragment>
         <Heading>CONTRACT</Heading>
-        <SubHeading onClick={() => setCurrentEditor('contract')}>Contract Template</SubHeading>
-        <SubHeading onClick={() => setCurrentEditor('contract')}>Contract Text</SubHeading>
+        <SubHeading onClick={() => setCurrentEditor(null, 'contract')}>Contract</SubHeading>
```

Comment:
Q: What is the first `null` parameter about?

Reply:
`setCurrentEditor` takes an `id` and an `editor type` as params. The `id` param is needed for clause templates so a designer can edit multiple clause templates and we can keep track of which one is currently being edited. In the case above, we are setting the current editor to the contract editor. Since there is only one overarching contract, we do not need an `id` in this case.